nuc_morph_analysis/analyses/height/toymodel.py

```diff
@@ -17,64 +19,77 @@
 plt.rcParams["font.family"] = "Arial"
 
 
-def toymodel(nc_ratio=0.29, cvol=0.5e6, num_workers=1):
+def toymodel(nc_ratio=0.29, cvol=0.5e6, cell_H_mod=0, num_workers=1, old_unfiltered_method=False):
     """
     main function to run
 
     args
     --------
     nc_ratio: ratio of nuclear to cytoplastic volumes - default is 0.3
-    cvol: target volume of nuclei to fit toy model to
+    cvol: target volume of nuclei to fit toy model to, default is 0.5e6 pixels^3 ~ 630 um^3
+    cell_H_mod: height difference between top of nucleus and top of cell to use for toy model. default is 0
     num_workers: how many workers to use for multiprocessing
+    old_unfiltered_method: whether to use the old (unfiltered) method that does not remove bad pseudo cells
```

Comment:
I think if folks dont have context for what is old it is confusing and github has the history of this in bioarchive branch so I would recommend we can still have an easy filter flag that is `filter_bad_density_measurements=True` would make more sense?